### DISCUSSION TITLE
Envisalink in Uno standalone mode

### DISCUSSION THREAD

**Discussion by pileofkyle** [ufodone/envisalink_new#62] [Q&A]
Would be it be possible to add support for this configuration? 
Thanks for all the work on this project!

**Comment by ufodone** [comment]
The Envisalink uses different protocols for Honeywell and DSC systems.  I don't know what the Uno uses.  It's probably a long shot but if it emulates one of those two protocols, then the integration might just work.  Have you tried configuring it as a DSC and/or Honeywell system?

If it has it's own protocol, then it's not likely something I'd be able to add as I don't have one of those units.

**Reply by michae1a1ee** [reply]
So does this envisalink_new not work with the EyezOn Uno panel?  The panel used DSC keypads and likely emulates the DSC alarm panels as it's made by former DSC employees.  EyezOn makes the Envisalink device and supposedly an Envisalink is built into the Uno panel according to what they've told me.

**Reply by ufodone** [reply]
Basic support exists which includes zone and partition state updates and simple arm/disarm functionality but it's not currently possible to arm in different mode (away, stay, etc.).  Limitations in the device firmware are currently preventing any additional features.  I'm told changes will be coming at some point to provide more complete support at which point I hope to update this integration to match what's available on the devices.

> The panel used DSC keypads and likely emulates the DSC alarm panels

The Uno panels actually use the Honeywell interface.  I'm not sure why that was chosen given they use DSC keypads.
